### PR TITLE
Remove trailing spaces and newlines

### PR DESCRIPTION
**Pull request by Flakebi**
This still leaves a single \n at the end of every file.

### CODE REVIEW

**Comment by amdvlk-admin** [comment]
Can one of the admins verify this patch?

File: llpc/lower/llpcSpirvLowerGlobal.cpp [hunk 1]
```diff
@@ -28,7 +28,6 @@
  * @brief LLPC source file: contains implementation of class Llpc::SpirvLowerGlobal.
  ***********************************************************************************************************************
  */
-
 #include "llvm/ADT/SmallSet.h"
```
Comment (nhaehnle) [inline llpc/lower/llpcSpirvLowerGlobal.cpp]: I'm not sure whether this particular changes is worth it, but it doesn't really hurt either...

Comment (Flakebi) [reply llpc/lower/llpcSpirvLowerGlobal.cpp]: I forgot to add that to #468, the include lines start directly after the comments everywhere else.
I could move it to the other patch if wanted.

Comment (nhaehnle) [reply llpc/lower/llpcSpirvLowerGlobal.cpp]: That would be good, yes.

**Comment by JacobHeAMD** [comment]
ok to test

**Comment by JacobHeAMD** [comment]
test this please

**Comment by amdvlk-admin** [comment]
### Test summary for commit 9000e2a564419164463099e0a83eb51f6823402f
<details open><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master b601c88aeca7a7b08becb3d32709de383c8ee428</li>
<li>METROHASH: amd-master 2b6fee002db6cc92345b02aeee963ebaaf4c0e2f</li>
<li>PAL: dev 9fab16015e522fff05890a045a1e9d8d3c23a636</li>
<li>SPVGEN: dev a61db14f75728e91c243d4bfc1c39700e9dc0ba6</li>
<li>XGL: dev 052ebfa2e77abd14fdb4f5120b35b081f3fc4bdc</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 1ecf0a1496f32eaf34825b3d11432c6b1d03c951</li>
</ul></details>

<details open><summary>ShaderDB test</summary><ul>

<li>Passed: 942/942 (100.0%)</li>
<li>Failed: 0/942 (0.0%)</li>
</ul></details>

<details open><summary>CTS tests (built with branch Vulkan-CTS-1.1.2.3-cherry-picks)</summary><ul>


<details open><summary>Rhel 7.6, Vega10</summary><ul>

<li>Passed: 2155/2156 (100.0%)</li>
<li><details><summary>Failed: 1/2156 (0.0%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.api.object_management.multithreaded_shared_resources.compute_pipeline
Stack trace: Timeout or crahsed
```
</p></details></li>


</ul></details>

<details open><summary>Ubuntu 18.04, Navi10</summary><ul>

<li>Passed: 65463/65463 (100.0%)</li>
<li>Failed: 0/65463 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Ellesmere</summary><ul>

<li>Passed: 65431/65431 (100.0%)</li>
<li>Failed: 0/65431 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by Flakebi** [comment]
retest this please

**Comment by amdvlk-admin** [comment]
### Test summary for commit 9000e2a564419164463099e0a83eb51f6823402f
<details open><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master b601c88aeca7a7b08becb3d32709de383c8ee428</li>
<li>METROHASH: amd-master 2b6fee002db6cc92345b02aeee963ebaaf4c0e2f</li>
<li>PAL: dev 9fab16015e522fff05890a045a1e9d8d3c23a636</li>
<li>SPVGEN: dev a61db14f75728e91c243d4bfc1c39700e9dc0ba6</li>
<li>XGL: dev 052ebfa2e77abd14fdb4f5120b35b081f3fc4bdc</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 1ecf0a1496f32eaf34825b3d11432c6b1d03c951</li>
</ul></details>

<details open><summary>ShaderDB test</summary><ul>

<li>Passed: 942/942 (100.0%)</li>
<li>Failed: 0/942 (0.0%)</li>
</ul></details>

<details open><summary>CTS tests (built with branch Vulkan-CTS-1.1.2.3-cherry-picks)</summary><ul>


<details open><summary>Rhel 7.6, Vega10</summary><ul>

<li>Passed: 65463/65463 (100.0%)</li>
<li>Failed: 0/65463 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Navi10</summary><ul>

<li>Passed: 65463/65463 (100.0%)</li>
<li>Failed: 0/65463 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Ellesmere</summary><ul>

<li>Passed: 65431/65431 (100.0%)</li>
<li>Failed: 0/65431 (0.0%)</li>
</ul></details>


</ul></details>

**Pull request merged**